articles/hooks/guides/create-delete-hooks-using-cli.md

```diff
@@ -6,20 +6,28 @@ topics:
     - cli
 contentType: how-to
 useCase: extensibility-hooks
+v2: true
 ---
+# Create/Delete Hooks Using the Command-Line Interface
```

Comment:
You could maybe break this into two separate docs "Create Hook Using the Command-Line Interface" and "Delete Hooks Using the Command-Line Interface."